### PR TITLE
Feat: Contribution Ways

### LINKED ISSUE #66: Coding: Develop Ways to Contribute Section of Contribute Page
## Description
As a user,
I need the following section of the mockup to be developed,
so that I can add it as Ways To Contribute Component [here](https://github.com/anitab-org/anitab-org.github.io/tree/develop/src/Components).

## Mocks
(Only the section in the red box to be developed)

Code-ways-to-contribute
Font Size for headings: Header Medium(32pt) weight- Medium
Font Size for the description below: body 18pt weight- Light
Font Style- Avenir
Font Color (heading): #103B81
Font Color (Body): #103B81

Margins given for Code section should be implemented for all other sections too,

[Design Link For Development](https://xd.adobe.com/view/b3ce723d-5bc6-45f5-7035-0a637d919751-c5ff/)

### Update [Required]
- [ ] Follows all the design guidelines as mentioned and is responsive enough on large screen devices

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
3-5 hours

### PR DESCRIPTION
**Pull request by tichnas**
### Description

Ways to Contribute section (on Contribute page) is added.
The code takes data dynamically from a file `contribution_ways.js`, but the real content is not yet added (#60)

Fixes #66 

### Type of Change:

- Code
- User Interface
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?

Project Name

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/content/contribution_ways.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        },
+        {
+          name: 'lorem',
+          url: 'https://anitab-org.zulipchat.com/#narrow/stream/212722-general',
+        },
+      ],
+      content: [
+        {
+          par:
+            'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Similique, incidunt veniam dolores id asperiores ad, rerum pariatur eligendi doloremque mollitia quas quis! Reiciendis maiores ab ullam accusantium consequatur numquam modi!',
```
Comment (annabauza) [inline src/content/contribution_ways.js:22]: No lorem impsum will be merged

**Review by annabauza — CHANGES_REQUESTED** [review]
Please replace Lorem ipsum with proper text or rise dependent issue for missing content. Thanks.

File: src/content/contribution_ways.js (hunk 1 above)
Comment (annabauza) [reply src/content/contribution_ways.js:22]: https://github.com/anitab-org/anitab-org.github.io/pull/181 it seems this PR is about content.

**Comment by nandini45** [comment]
@tichnas  can you tell me what changes do u need me to make in my code for the content of this page?

**Comment by tichnas** [comment]
@nandini45 Thanks for asking. I want the file to be named as `contribution_ways.js` (instead of `contribution_ways.json`). Other than that, it should follow the structure as given here: https://github.com/anitab-org/anitab-org.github.io/blob/3e97ad69f81978b39779d1cdc4be75ed87c780c9/src/content/contribution_ways.js

Some random text is there to get an idea of what to put where, but please let me know if something is not clear.

**Comment by nandini45** [comment]
@tichnas  i am a little caught up with my work
 can you do the work?
 i will assign the issue to u.
 you can take my content reformat it the way u want and open a PR
 thank you

**Comment by tichnas** [comment]
@nandini45 Ok, no issues

**Comment by Amulya-coder** [comment]
@tichnas Can I work on this issue.

**Comment by tichnas** [comment]
Hi @Amulya-coder, thanks for your interest in contributing to the community but I already have a PR open for this issue.

**Comment by tichnas** [comment]
@Amulya-coder By this issue do you mean https://github.com/anitab-org/anitab-org.github.io/issues/60?

**Comment by Amulya-coder** [comment]
@tichnas Yes ,I want to work on this section but I am pretty new to React-Native. If you don't mind , I can take a little bit more time to develop this than the required time.

**Comment by Vishal-raj-1** [comment]
@tichnas Please resolve conflicts !!

**Comment by annabauza** [comment]
@tichnas I close this PR due to lack of respond.

**Pull request closed without merging**